example/app/config.yml

```diff
@@ -81,17 +81,38 @@ default:
     subnets:
       - '{{AWS_SUBNET}}'
 
+  alarms:
```

Comment:
So these alarms are what actually trigger the scaling when the CPU usage goes above/below a certain threshold?

Reply:
yep